perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1723,6 +1729,10 @@ sub save_user_and_report : Private {
     } elsif ($c->stash->{contributing_as_anonymous_user}) {
         $report->set_extra_metadata( contributed_as => 'anonymous_user' );
         $report->confirm();
+    } elsif ($c->stash->{ignore_logged_in_user}) {
+        # Waste report on behalf of customer
+        $report->user->update_or_insert;
```

Comment:
We don't want to e.g. update a user account's name if the staff have given it a different name.